### ISSUE TITLE
RFE: Ignore scratch builds if Packit config is present

### ISSUE THREAD

**Issue by penguinpee** [fedora-infra/the-new-hotness#591]
NeuroFedora SIG is adding more and more packages it maintains collectively to Packit. Before Packit we relied on scratch builds triggered by Anitya on retrieving a new release.

It seems we often forget to change the monitoring setting in dist-git from "scratch builds" to "monitoring" after having added a Packit config file (`.packit.yml` or `.packit.yaml`) to git. One reason is probably the fact that the setting can only be changed by an admin. For most packages the access for neuro-sig, the NeuroFedora group, is limited to commit access, which is more than sufficient.

So, my idea is to have Anitya be smart about it and ignore monitoring status "scratch builds" if it finds a Packit config file in the packages repo. We'd still like to have bugs opened for tracking purposes. Maybe Anitya could add a comment like:

_Skipping scratch build due to presence of Packit config. Please consider setting monitoring status to "monitoring"._

**Comment by Zlopez** [comment]
I transferred this to the-new-hotness as Anitya is not the one who is interacting with dist-git. 

I'm not sure if just checking for the packit config file is a good way to do it as it doesn't need to mean that the scratch builds are done by packit. But I will look into this and try to find some solution. We should probably think of something together with packit folks as the-new-hotness and packit are sometimes overlapping in their responsibilities.

**Comment by penguinpee** [comment]
Thanks for the transfer. I always seem to get this wrong[^1].

I thought some more about it and actually came to the same conclusion. Packit is only submitting a PR. But it depends on the package being configured for CI and/or Zuul if a scratch build will be triggered. Thus, there might be packages still relying on the scratch builds triggered by hotness.

I'll leave it to you and the Packit folks to figure out what's sensible here. Maybe the function of the button can be changed or extended or a new button added? Just a thought.

[^1]: At least there is consistency in my failures. :wink:

**Comment by Zlopez** [comment]
Let me look in the packit issue and we try to figure something out.

**Comment by sergiomb2** [comment]
if we go from  "scratch builds" to "monitoring" , will the-new-hotness open a bugzilla report ?

**Comment by Zlopez** [comment]
@sergiomb2 Yes, it will just not start scratch build. See https://the-new-hotness.readthedocs.io/en/stable/user-guide.html#notifications-settings
